### PR TITLE
added battery status for linux and freebsd (only one battery)

### PR DESCRIPTION
**Pull request by dawidd6**
zrzut ekranu z 2016-03-20 15-31-57
It's showing now `Charging` (linux) or `charging` (FreeBSD). Works only for one battery, i cannot test it for multiple batteries, so i didn't touched that.

### CODE REVIEW

**Comment by iandrewt** [comment]
If you're detecting the charging status to determine whether to return the charging status, you don't need to use the charging status variable to print it, you can just say "Charging". This would also keep the case of the word consistent between the Linux and FreeBSD. I'll add OS X support and put it in your pull request too.

**Comment by dylanaraps** [comment]
For multiple battery support we may be able to do something like this:

```sh
cat /sys/class/power_supply/BAT${battery_num}/capacity /sys/class/power_supply/BAT${battery_num}/status
```

**Comment by dylanaraps** [comment]
This should group each battery percentage to its status.

**Comment by iandrewt** [comment]
I've added OS X support and made a pull request to be pulled into @dawidd6's branch

File: neofetch [hunk 1]
```diff
@@ -1551,7 +1551,8 @@ getbattery () {
 
                 # Create an array of battery usage from each battery.
                 batteries=($(cat /sys/class/power_supply/BAT${battery_num}/capacity))
-
+				# Get the power status
```
Comment (dylanaraps) [inline neofetch]: Fix the indentation here.

File: neofetch [hunk 2]
```diff
@@ -1551,7 +1551,8 @@ getbattery () {
...
+				power_status=($(cat /sys/class/power_supply/BAT${battery_num}/status))
```
Comment (dylanaraps) [inline neofetch]: Fix the indentation here.

File: neofetch [hunk 3]
```diff
@@ -1578,8 +1579,11 @@ getbattery () {
                         done
                         return
                     fi
-
-                    battery="${batteries[0]}%"
+						if [ $power_status = "Charging" ]; then
```
Comment (dylanaraps) [inline neofetch]: Should be `if [ "$power_status" == "Charging" ]; then`.

Also, fix the indentation here.

File: neofetch [hunk 4]
```diff
@@ -1578,8 +1579,11 @@ getbattery () {
...
+		                battery="${batteries[0]}% Charging"
```
Comment (dylanaraps) [inline neofetch]: Indentation.

File: neofetch [hunk 5]
```diff
@@ -1578,8 +1579,11 @@ getbattery () {
...
+		                else
```
Comment (dylanaraps) [inline neofetch]: Indentation.

File: neofetch [hunk 6]
```diff
@@ -1578,8 +1579,11 @@ getbattery () {
...
+		                battery="${batteries[0]}%"
```
Comment (dylanaraps) [inline neofetch]: Indentation.

File: neofetch [hunk 7]
```diff
@@ -1578,8 +1579,11 @@ getbattery () {
...
+		                fi
```
Comment (dylanaraps) [inline neofetch]: Indentation.

File: neofetch [hunk 8]
```diff
@@ -1589,7 +1593,13 @@ getbattery () {
         "BSD")
             case "$distro" in
                 "FreeBSD"*)
+            		power_status=$(acpiconf -i 0 | awk -F ':\t\t\t' '/State/ {print $2}')
```
Comment (dylanaraps) [inline neofetch]: Indentation.

File: neofetch [hunk 9]
```diff
@@ -1589,7 +1593,13 @@ getbattery () {
...
+            		
+            		if [ $power_status = "charging" ]; then
```
Comment (dylanaraps) [inline neofetch]: Should be `if [ "$power_status" == "Charging" ]; then`

Also, indentation.

**Comment by dylanaraps** [comment]
Yo, I've left some comments. 

Some more notes:

- Rename `power_status` to `battery_status`.
- Lets polish this for single battery initially and we'll work on multi later.

Good job so far!

**Comment by dawidd6** [comment]
Yeah, it's a quite strange situation. On github indentations are different than in gedit.

Seems like nano is closer to github. Which editor are you using?

**Comment by dylanaraps** [comment]
I'm using vim/neovim.

**Comment by dawidd6** [comment]
OK, done. `nano` did the trick

File: neofetch [hunk 10]
```diff
@@ -1589,7 +1594,13 @@ getbattery () {
         "BSD")
             case "$distro" in
                 "FreeBSD"*)
-                    battery=$(acpiconf -i 0 | awk -F ':\t' '/Remaining capacity/ {print $2}')
+            	    battery_status=$(acpiconf -i 0 | awk -F ':\t\t\t' '/State/ {print $2}')
+
+            	    if [ "$battery_status" == "charging" ]; then
```
Comment (dylanaraps) [inline neofetch]: Change this `if` block to:

```sh
[ "$battery_status" == "charging" ] && \
    battery="${battery} Charging"
```

File: neofetch [hunk 11]
```diff
@@ -1589,7 +1594,13 @@ getbattery () {
         "BSD")
             case "$distro" in
                 "FreeBSD"*)
-                    battery=$(acpiconf -i 0 | awk -F ':\t' '/Remaining capacity/ {print $2}')
```
Comment (dylanaraps) [inline neofetch]: Put this line here:

```sh
battery=$(acpiconf -i 0 | awk -F ':\t' '/Remaining capacity/ {print $2}')
```

**Comment by dylanaraps** [comment]
You still have some indentation issues and I left some more comments.

**Comment by dylanaraps** [comment]
I'm going to submit a PR to this branch that makes some changes that are a little hard to explain here.

**Pull request merged**